src/_base/harness/config/functions.yml

```diff
@@ -2,6 +2,10 @@ function('to_yaml', [text]): |
   #!php
   = preg_replace('/^/m', '  ', \Symfony\Component\Yaml\Yaml::dump($text, 100, 2));
 
+function('to_nice_yaml', [text, indentation, nesting]): |
+  #!php
+  = preg_replace('/^/m', str_repeat(' ', $nesting ?: 2), \Symfony\Component\Yaml\Yaml::dump($text, 100, $indentation ?: 2));
+
```

Comment:
added as another function as there is no current method to force a ws tool version upgrade, and people will be using versions where optional parameters aren't supported